Offices/Offices.MessageBroking/DependencyInjection/ConsumersDependencyInjection.cs

```diff
@@ -0,0 +1,77 @@
+﻿using MassTransit;
+using Microsoft.Extensions.DependencyInjection;
+using Offices.Consumers.Mapper;
+using Offices.MessageBroking.Consumers.DoctorConsumers;
+using Offices.MessageBroking.Consumers.ReceptionistConsumers;
+using Offices.MessageBroking.Mapper;
+using Offices.MessageBroking.Producers;
+using Offices.MessageBroking.Producers.Abstractions;
+
+namespace Offices.MessageBroking.DependencyInjection
+{
+    public static class ConsumersDependencyInjection
+    {
+        public static IServiceCollection AddConsumersDependencyInjection(this IServiceCollection services)
```

Comment:
Why is it called 'consumers' dependency injection if you have both consumers and producers here